documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -2262,6 +2262,19 @@ Again, regularly emitting events solves the problem.
 ====
 endif::community[]
 
+
+// Type: concept
+// ModuleID: postgresql-deploying-multiple-connectors
+// Title: Setting up multiple connectors for same database server
+[[postgresql-deploying-multiple-connectors]]
+=== Setting up multiple connectors for same database server
+
+It is highly possible that a single Postgres cluster hosts multiple databases (e.g. if the cluster is shared among multiple services, each service has its own database), since a replication slot only streams data from a single database, you'll need a replication slot for each database, or rather for each connector. As even if two connectors are pointing to the same database, they may have different table filters and other configs. To avoid them interfering with each other and thus causing unexpected behavior, it's best to keep unique slot for each connector - the general idea being to keep one replication slot per consumer.
```

Comment:
I would reword this because each connector requires its own unique replication slot regardless of the number of databases hosted by the PG cluster.  Keep in mind, a slot is effectively a pointer into the WAL and each connector can be consuming from different positions, and needs to independently manage that pointer.  Furthermore, PG doesn't even allow you to use a replication slot if its already active, i.e. in use by another process.